sbhs_server/experiment/views.py

```diff
@@ -3,76 +3,47 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth import authenticate
 from django.contrib.auth import login as LOGIN
-from sbhs_server.tables.models import Slot, Account, Experiment, Booking
+from sbhs_server.tables.models import Account, Experiment
 import json, datetime, os, time
 from django.views.decorators.csrf import csrf_exempt
-from sbhs_server import settings
+from sbhs_server import settings,sbhs
 # Create your views here.
 # 
+
+global boards
```

Comment:
'boards' is a Sbhs object that is created when the user logs in from the client application in the initiation() method. I've declared it global as the object created during initiation() is being used in the experiment() method.